### ISSUE TITLE
Consider updating default bandit

### ISSUE THREAD

**Issue by jbosboom** [jansel/opentuner#49]
The default bandit technique (AUCBanditMetaTechniqueA) contains NormalGreedyMutation, UniformGreedyMutation, RandomNelderMead and DifferentialEvolutionAlt.  From personal communication with Jason, this bandit was chosen somewhat arbitrarily.  We've added more techniques since then, so we should consider changing this default, or at least justifying it with some data.

**Comment by jansel** [comment]
I agree with this, we should definitely do a survey of technique performance across different benchmarks to improve our defaults.
